iphone/cli/commands/_build.js

```diff
@@ -7102,6 +7102,8 @@ iOSBuilder.prototype.invokeXcodeBuild = async function invokeXcodeBuild(next) {
 			this.logger.warn(`The app is using native modules (${Array.from(this.legacyModules)}) that do not support arm64 simulators, we will exclude arm64. This may fail if you're on an arm64 Apple Silicon device.`);
 			args.push('EXCLUDED_ARCHS=arm64');
 		}
+	} else if ( this.target === 'device' || this.target === 'dist-adhoc' ){
```

Comment:
Does this also need to include the `dist-appstore` target or is the error not seen when targeting that?

Reply:
Also, eslint will complain about the spaces in the brackets. If you have this checked out locally you can run `npm run format:js` to fix them and any other issues automatically, you might need to run `npm ci` first.

Reply:
No need to include dist-appstore. 
Probably because appstore provisioning profiles don't include devices